cli/tests/unit_node/buffer_test.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import { Buffer } from "node:buffer";
+import { assertEquals } from "../../../test_util/std/testing/asserts.ts";
+
+const buffer_len12 = Buffer.from("Hello, World!");
+const buffer_len11 = Buffer.from("Hello, World");
+
+const BUFFER_ENCODINGS = [
+  "hex",
+  "base64",
+  "base64url",
+  "ascii",
+  "latin1",
+  "ucs2",
+  "utf8",
+  "utf16le",
+];
+
+for (let encoding of BUFFER_ENCODINGS) {
+  Deno.test({
+    name: "Buffer.from with encoding " + encoding,
+    fn: () => {
+      // @ts-ignore
```

Comment:
Why ts-ignore here?

Reply:
Otherwise I get this:

```
TS2769 [ERROR]: No overload matches this call.
   Overload 1 of 4, '(arrayBuffer: WithImplicitCoercion<ArrayBuffer | SharedArrayBuffer>, byteOffset?: number | undefined, length?: number | undefined): Buffer', gave the following error.
     Argument of type 'string' is not assignable to parameter of type 'WithImplicitCoercion<ArrayBuffer | SharedArrayBuffer>'.  Overload 2 of 4, '(str: WithImplicitCoercion<string> | { [globalThis.Symbol.toPrimitive](hint: "string"): string; }
 , encoding?: BufferEncoding | undefined): Buffer', gave the following error.
     Argument of type 'string' is not assignable to parameter of type 'BufferEncoding | undefined'.
       assertEquals(Buffer.from(buffer_len11.toString(encoding), encoding), buffer_len11);
```

Which is weird because BufferEncoding is a string.

Reply:
Try adding `as BufferEncoding[]` on line 17

Reply:
BufferEncoding is not exported by node:buffer so let's just use ts-ignore for now